packages/story-editor/src/components/inspector/inspectorProvider.js

```diff
@@ -36,11 +36,12 @@ import { useFeature } from 'flagged';
 import { useAPI } from '../../app/api';
 import { useStory } from '../../app/story';
 import { useHighlights } from '../../app/highlights';
-import { DOCUMENT, DESIGN, PUBLISH_MODAL_DOCUMENT } from './constants';
+import { DOCUMENT, STYLE, PUBLISH_MODAL_DOCUMENT, INSERT } from './constants';
 import Context from './context';
 import DesignInspector from './design';
+import Library from '../library';
```

Comment:
When things are renamed, folders should also be moved around accordingly. Either the library should go inside the inspector (to be renamed) folder or both document and design panels should move out. I feel like the latter is the proper approach. But that's work for a future refactor.

Reply:
+1 to making this refactor and doing it as its own ticket.